src/Graph.js

```diff
@@ -457,13 +459,14 @@ class DemandSupplyGraphAUC extends DemandSupplyGraph {
         ], invisiblePointOptions);
 
         const points = [p3, p1, p2];
-        this.board.create('polygon', points, {
+        const p = this.board.create('polygon', points, {
             withLabel: false,
             fillColor: 'red',
             highlightFillColor: 'red',
             ...triangleOptions
         });
         drawLabel(this.board, points, 'C');
+        this.triangleCArea = forceFloat(p.Area());
```

Comment:
To make this in a more stateless, functional programming style, I would just `return p;` at the end of this function. Then in `make` below, call `triangleA.Area()` as needed.